server.go

```diff
@@ -36,6 +36,11 @@ import (
 	"github.com/globalsign/mgo/bson"
 )
 
+const (
+	defaultWriteBatchSize      = 1000
+	defaultMaxMessageSizeBytes = 48000000
```

Comment:
The max message size is not verified anywhere in the code.

Reply:
It is in `sendMessage()` [here](https://github.com/globalsign/mgo/pull/61/files#diff-a75970bbdfa9237a3958512f8cc2fb09R634)

Reply:
Damn. I think I have missed a whole part of the PR.